lib/puppet/provider/cloudwatch_alarm/v2.rb

```diff
@@ -33,10 +33,15 @@ def self.prefetch(resources)
   read_only(:region, :alarm_actions)
 
   def self.alarm_to_hash(region, alarm)
-    response = autoscaling_client(region).describe_policies(
-      policy_names: alarm.alarm_actions
-    )
-    actions = response.scaling_policies.collect(&:policy_name)
+    actions = alarm.alarm_actions
+
+    unless actions.first.start_with?('arn:aws')
```

Comment:
Hey, yes, we should probably inspect all actions. What should the code do in this case? Collect all the actions that are not arn as pass as parameter  to describe_policies? Then merge "actions" back?

I can work on that if we agree with this solution. Sorry for taking almost one month to respond :(